File: back/src/main/java/csh/back/domain/vote/vote/dto/response/VoteFindUserResponse.java
package csh.back.domain.vote.vote.dto.response;

import csh.back.domain.vote.user.entity.VoteUser;
import csh.back.domain.vote.vote.entity.Vote;

public record VoteFindUserResponse(String name) {
    public static VoteFindUserResponse from(VoteUser voteUser) {
        return new VoteFindUserResponse(voteUser.getTripMember().getMember().getName());
    }
}

File: back/src/main/kotlin/csh/back/domain/vote/vote/dto/response/VoteFindUserResponse.kt
package csh.back.domain.vote.vote.dto.response

import csh.back.domain.vote.user.entity.VoteUser

data class VoteFindUserResponse(
    val name: String,
) {
    companion object {
        @JvmStatic
        fun from(voteUser: VoteUser): VoteFindUserResponse =
            VoteFindUserResponse(voteUser.tripMember.member.name)
    }
}
